config/defines.inc.php

```diff
@@ -68,16 +68,29 @@
 if (defined('_PS_ADMIN_DIR_')) {
     define('_PS_BO_ALL_THEMES_DIR_', _PS_ADMIN_DIR_.'/themes/');
 }
-if (!defined('_PS_CACHE_DIR_')) {
-    if (defined('_PS_IN_TEST_')) {
-        $env = 'test';
-    } else {
-        $env = (_PS_MODE_DEV_) ? 'dev' : 'prod';
+
+// Find if we are running under a Symfony command
+$cliEnvValue = null;
```

Comment:
lets wrap this into a class with a public static method instead ?

Reply:
Not possible because we don't have access to the PrestaShop autoload at this moment